clients/admin-ui/src/features/common/system-data-flow/DataFlowAccordionForm.tsx

```diff
@@ -0,0 +1,193 @@
+import {
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Button,
+  ButtonGroup,
+  Flex,
+  Spacer,
+  Stack,
+  Tag,
+  Text,
+  useDisclosure,
+  useToast,
+} from "@fidesui/react";
+import { FormGuard } from "common/hooks/useIsAnyFormDirty";
+import { GearLightIcon } from "common/Icon";
+import QuestionTooltip from "common/QuestionTooltip";
+import { DataFlowSystemsDeleteTable } from "common/system-data-flow/DataFlowSystemsDeleteTable";
+import DataFlowSystemsModal from "common/system-data-flow/DataFlowSystemsModal";
+import { successToastParams } from "common/toast";
+import { Form, Formik } from "formik";
+import React, { useEffect, useMemo, useState } from "react";
+
+import {
+  useGetAllSystemsQuery,
+  useUpdateSystemMutation,
+} from "~/features/system";
+import { DataFlow, System } from "~/types/api";
+
+const defaultInitialValues = {
+  dataFlowSystems: [] as DataFlow[],
+};
+
+export type FormValues = typeof defaultInitialValues;
+
+type DataFlowAccordionItemProps = {
+  isIngress?: boolean;
+  system: System;
+  isSystemTab?: boolean;
+};
+
+export const DataFlowAccordionForm = ({
+  system,
+  isIngress,
+  isSystemTab,
+}: DataFlowAccordionItemProps) => {
+  const toast = useToast();
+  const flowType = isIngress ? "Source" : "Destination";
```

Comment:
this should be lowercase. or the part that renders "Configure {source | destination} " (at least that's what I'm seeing in the design. in general, I've been trying to sentence case everything)

Reply:
I only lowercased it in that one spot since the uppercase version is needed in a couple other spots